### ISSUE TITLE
Taskbar with Fedora26 test and with btrfs locks up taskbar and locks up the system.

### ISSUE THREAD

**Issue by lsatenstein** [zpydr/gnome-shell-extension-taskbar#154]
TB 56, Gnome 3.24
With /home/tbuser on ext4 and / on ext4, all seems to be fine.  (every partition of Fedora26 based on ext4)

With /home/TBuser on ext4 and / on btrfs  some installation tests with gnome.org crash and results in loss of the keyboard (no keyboard activity recognized. and also lose the setting gnome.org.
/with /home/TBuser on btrfs and / on btrfs clicking on the install either locks the system hard, or clicking on any setting (turn on favourites, align, etc.) locks up the system.

Debugging  as per debugging guide.
cleaned ./local/share/gnome-shell/extensions/TaskBar 
also verified that /usr/share/gnome-extensions.... does not include TaskBar
Did one test with TaskBar alone -- failed with all the above choices
Did one test with another extension (OK) and with Taskbar. Same as the "stand-alone test"

I am not sure if you are exercising a part of Gnome that was not QA'd or if there is a problem with Gnome and btrfs or with TaskBar and btrfs.

My email is lsatenstein@yahoo.com.  Want me to test a version with debugging therein, let me know.
or send me a link from where I can retrieve a diagnostic version of Taskbar.  
I have no issues with TaskBar and Fedora 25 (ext4 partition formats)

**Comment by zpydr** [comment]
Fedora 26 is beta software, so there could still be a few issues with that OS version.
I cannot replicate this issue with btrfs/ext4, also there shouldn't be any relation with using TaskBar.
There is no diagnostic version of TaskBar, the latest available version is available here on github.
Emails to lsatenstein@yahoo.com return undelivered.

**Comment by zpydr** [comment]
I tried reproducing this issue, but it is working fine with btrfs and ext4.

**Comment by lsatenstein** [comment]
I think it is a Fedora 26 beta problem and their user of Gnome 3.24  Thank you for following through.
I spend many hours trying to identify and resolve the problem.  Taskbar always installed, but when I used the align option, that was it.   Once I got it setup, (bottom center), there was no longer an issue. Only issue was with configuration.
Thank you for such a great product. Without your Taskbar, I would have abandoned my use of Gnome.
 
Regards 
 Leslie
 Leslie Satenstein
Montréal Québec, Canada




On Friday, June 30, 2017, 8:06:24 PM GMT-4, zpydr <notifications@github.com> wrote:


I tried reproducing this issue, but it is working fine with btrfs and ext4.
